CONTRIBUTING.md

```diff
@@ -0,0 +1,67 @@
+# How to Contribute
+
+Thanks for your interest in Noda Time. We appreciate all kinds of contributions, from submitting issues to improving documentation; from writing tests to implementing new code. All help is welcome!
+
+## Basic Requirements
+
+If you want to contribute to the codebase, you must have Visual
+Studio 2017 installed - you can download the Community edition from
+[here.](https://www.visualstudio.com/en-us/downloads/download-visual-studio-vs.aspx) (The other editions will work fine as well.)
+
+You're also going to need .NET Core SDK installed - you can download it from [here.](https://www.microsoft.com/net/download/)
+
+Please make sure you have a [git client](https://git-scm.com/) installed. If you don't already have a Github account, [please create one.](https://github.com/signup/free)
+
+After you're all set, you can [fork the project](https://help.github.com/articles/fork-a-repo). Then you'll be able to clone your fork, so you can edit the files locally on your machine:
+
+`git clone https://github.com/YOUR-USERNAME/nodatime.git`
+
+## How to start contributing?
+
+We have a [`help-wanted`](https://github.com/nodatime/nodatime/labels/help%20wanted)
+label on our issue tracker to indicate tasks which contributors can work on.
```

Comment:
Maybe "to indicate tasks which contributors can work on without much previous experience in Noda Time"?

Reply:
Sounds great!